numpy/_core/src/umath/string_ufuncs.cpp

```diff
@@ -541,6 +897,22 @@ init_string_ufuncs(PyObject *umath)
         return -1;
     }
 
+    if (init_isdigit(umath) < 0) {
+        return -1;
+    }
+
+    if (init_isspace(umath) < 0) {
+        return -1;
+    }
+
+    if (init_isdecimal(umath) < 0) {
+        return -1;
+    }
+
+    if (init_isnumeric(umath) < 0) {
+        return -1;
+    }
```

Comment:
I can live with 4 calls here.  But there is no reason that it isn't the same function that gets called.